### PR TITLE
message versioning

### PR DESCRIPTION
**Pull request by soonkuk**
### Github Issue
Resolves ##31

### Background
Ballot and Transaction message Versioning.

### CODE REVIEW

File: lib/common/version.go [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package common
+
+const (
+	TransactionMessage_Version1 = "tx_version1"
```
Comment (spikeekips) [inline lib/common/version.go]: Please remove `_`, `TransactionMessage_Version1` -> `TransactionMessageVersion1`. How about remove `Message` in name? Just `TransactionVersion`, `Version1` does not mean anything, I think.

File: lib/common/version.go [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+	BallotMessage_Version1      = "ballot_version1"
+)
```
Comment (spikeekips) [inline lib/common/version.go]: @soonkuk We talked about version, the version value should be `1`, not `tx_version1`

File: cmd/sebak/cmd/key/verify [hunk 3]
```diff
@@ -0,0 +1,131 @@
+package key
```
Comment (spikeekips) [inline cmd/sebak/cmd/key/verify]: @soonkuk What is this? Is it related to this PR?

File: cmd/sebak/cmd/wallet/payment.go [hunk 4]
```diff
@@ -189,6 +189,7 @@ func makeTransactionCreateAccount(kpSource keypair.KP, kpDest keypair.KP, amount
 
 	tx := transaction.Transaction{
 		H: transaction.Header{
+			Version: common.TransactionVersionV1V1,
```
Comment (spikeekips) [inline cmd/sebak/cmd/wallet/payment.go]: `TransactionVersionV1V1` is right? `TransactionVersionV1`?

File: lib/common/message.go [hunk 5]
```diff
@@ -6,9 +6,11 @@ import (
 )
 
 const (
-	ConnectMessage     MessageType = "connect"
-	TransactionMessage MessageType = "transaction"
-	BallotMessage      MessageType = "ballot"
+	ConnectMessage       MessageType = "connect"
+	TransactionMessage   MessageType = "transaction"
+	BallotMessage        MessageType = "ballot"
```
Comment (Charleslee522) [inline lib/common/message.go]: New line is better for distinguish between `MessageType` and `Version`

File: cmd/sebak/cmd/key/verify (hunk 3 above)
Comment (soonkuk) [reply cmd/sebak/cmd/key/verify]: @spikeekips Oh it's my mistake. I'll remove that file.

**Pull request merged**